apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+ 
+## Setup the project and environment  
+
+Now, we will setup the project and environment that we will create therein. In the first stage, we will access the cloud portal and create a new *project*. A project is connected to a source code repository, for example, GitHub.  
+
+![A GitHub repository is mapped to a project in XM Cloud](https://sitecorecontenthub.stylelabs.cloud/api/public/content/142a8648763a42eda1a210b1488cfda5?v=859d5477) 
+
+Each project can have multiple environments. A typical setup is to have one source code repository for one Brand or legal entity, and then have a DEV environment, QA, Staging or Pre Prod, and Production. 
+
+![XM Cloud Projects and Environments](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9deae49bb7f5428285baf2ea374fe1ec?v=40ffa388)
+ 
+We will start with a Dev Environment for now, and create the other environments later. Let’s get back to project creation! 
+
+
+Let’s get started! 
+
+### Log into Sitecore Cloud Portal 
+
+1. To begin, open the Sitecore Cloud Portal (https://portal.sitecorecloud.io) and log in.   
+2. Access the XM Cloud project and environment management interface by clicking `XM Cloud Deploy`.  
+![Sitecore Portal - Open XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/42edc85bc7bd41e2914f29b7b260496b?v=94261954)
+
+### Create a new Project and Authoring Environment 
+
+1. From there, create a new project with the XM Cloud Deploy App by clicking `Create project`.  
+![Project Overview - Create new Project](https://sitecorecontenthub.stylelabs.cloud/api/public/content/2215cae974524bcb820ce28fd7fe3c51?v=df12ee70)  
+2. From here you provide a Project Name e.g.”XM Cloud Tutorial Series” and click the Continue button 
+![Create Project and Environment Step 1 - Provide Project Name](https://sitecorecontenthub.stylelabs.cloud/api/public/content/95a7fa3d5e7e4f6997494cae9a5ac70e?v=3678a3bd) 
+3. Choose whether you want Connect to GitHub or to Azure DevOps. A starter solution will be copied to your connected source code repository as a starting point. For the sake of this tutorial you choose GitHub and press the Continue Button   
+![Create Project and Environment Step 2 - Choose Source Code Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/27f501fc072d4a23b2a89a64b6e936f5?v=eafd675a) 
+
+In step 3 you can choose whether you want to start from the XM Cloud template or if you want to use your own code. The XM Cloud template is a starter kit based on Next.js, that contains already a rich set of features and sample implementations. It is best practice to start from the template.   
+
+4. Choose a GitHub Account from the dropdown or Connect to a new account by following the steps. Make sure you give rights to create a new repository in your account. 
```

Comment:
`Connect` should be `connect` (lowercase)

Reply:
It occurs to me that this is a reference to the button in the screenshot. I have updated the formatting and text to make this clear that the `Connect to a new account` is a button the user should click.

Reply:
Resolved in the latest commit